### PR TITLE
Feat: play button

### PR DESCRIPTION
**Pull request by bbucsek**
add style to the playsong component
add tests

### CODE REVIEW

File: src/components/PlaySongs/styles.ts [hunk 1]
```diff
@@ -41,7 +24,29 @@ export const Container = styled.div`
...
+    padding: 0 1rem;
+    background-color: ${({ theme }) => theme.colors.secondary};
+    border-radius: 10px;
+    border: 0;
+    font-size: 1rem;
+    color: ${({ theme }) => theme.colors.whiteFontColor};
+    cursor: pointer;
+    box-shadow: ${({ theme }) => theme.boxShadow};
+    margin: 1rem;
+
```
Comment (KlaraHajdu) [inline src/components/PlaySongs/styles.ts:44]: If you add: 
```
&:focus {
        outline: none;
    }
```

then the black outline will disappear, when clicked.

File: src/components/PlaySongs/styles.ts [hunk 2]
```diff
@@ -63,19 +68,27 @@ export const Title = styled.div`
...
+        transform: scale(1.1);
+    }
 `
+
 export const ButtonCanBeDisabled = styled.button<{disabled: boolean}>`
+    
     svg {
-        color: ${({ theme, disabled }) => disabled === true? "grey": theme.colors.blue};
+        color: ${({ theme, disabled }) => disabled === true ? theme.colors.disabled: theme.colors.blue};
+        cursor: ${({ theme, disabled }) => disabled === true ? "not-allowed" : "pointer"};
```
Comment (KlaraHajdu) [inline src/components/PlaySongs/styles.ts:95]: I would personally prefer not adding this 'not-allowed' value here, as it looks a bit strange to me. The changing colours already convey the message. But it is up to you.

**Review by KlaraHajdu — CHANGES_REQUESTED** [review]
Nice!

File: src/components/PlaySongs/styles.ts (hunk 2 above)
Comment (bbucsek) [reply src/components/PlaySongs/styles.ts:95]: i want to imitate the standard button disabled property

File: src/components/PlaySongs/styles.ts (hunk 1 above)
Comment (bbucsek) [reply src/components/PlaySongs/styles.ts:44]: fixed

**Review by KlaraHajdu — APPROVED** [review]

**Pull request merged**